ui/app/scripts/stream/controllers/definitions.js

```diff
@@ -41,6 +50,7 @@ define(['model/pageable'], function (Pageable) {
                 $scope.pageable.items = result._embedded.streamDefinitionResourceList;
               }
               $scope.pageable.total = result.page.totalElements;
+              loadMetrics();
```

Comment:
* We have a pageable list of runtime apps which only constitutes a subset of the data but we load the metrics un-parameterized. Shouldn't that ideally take the `pageable` into account? Or maybe even better should the metrics information be loaded only on mouse-over?
* We load the metrics without using the `BusyPromise`. In case the loading takes a long time, the user does not have a visual hue.